add general rac_aspace description to README (#89)
RockefellerArchiveCenter/rac_aspace: README.md

# rac_aspace

> `rac_aspace` is a Python package of common functions based on the Rockefeller Archive Center’s previous coding efforts with the ArchivesSpace API . The goal of the package is to improve code maintainability and facilitate code reuse across our organization. The package implements consistent linting and unit tests and builds upon existing Python packages such as ArchiveSnake. The built in functionality includes data helpers, serialization, and deleting and saving in ArchiveSpace (see descriptions below).

Reviewer: ```suggestion
`rac_aspace` is a Python package which is intended to supplement [ArchivesSnake](https://github.com/archivesspace-labs/ArchivesSnake) by providing common functions based on the Rockefeller Archive Center’s previous coding efforts with the ArchivesSpace API. The goal of the package is to improve code maintainability and facilitate code reuse across our organization. The built in functionality includes data helpers, serialization, and deleting and saving in ArchiveSpace (see descriptions below).
```
